source/community/reactnative/src/components/inputs/input.js

```diff
@@ -206,9 +206,10 @@ export class Input extends React.Component {
 			return null;
 		}
 		else {
+			let opacityStyle = { opacity: inlineAction.isEnabled == undefined ? 1.0 : inlineAction.isEnabled ? 1.0 : 0.4 };
```

Comment:
`undefined` check is not required as it's handled in **` BaseActionModel`**. can we remove this check?

Reply:
inlineAction inside the payload is not inherited from the BaseActionModel. Only ActionModel is inherited from the BaseActionModel. So we need to handle the null cases in the places where we consumed.